broker/src/main/java/io/zeebe/broker/exporter/metrics/MetricsExporter.java

```diff
@@ -50,7 +50,7 @@ public MetricsExporter() {
   public void open(final Controller controller) {
     this.controller = controller;
 
-    controller.scheduleTask(TIME_TO_LIVE, this::cleanUp);
+    controller.scheduleCancellableTask(TIME_TO_LIVE, this::cleanUp);
```

Comment:
Shouldn't we be saving the task? What happens if we close without cancelling the task? Goes for the other usage.